koku/masu/database/aws_report_db_accessor.py

```diff
@@ -250,7 +260,10 @@ def populate_ocp_on_aws_cost_daily_summary_trino(
 
         unattributed_storage = is_feature_unattributed_storage_enabled_aws(self.schema)
 
-        sql = pkgutil.get_data("masu.database", "trino_sql/reporting_ocpawscostlineitem_daily_summary.sql")
+        sql_file = "trino_sql/reporting_ocpawscostlineitem_daily_summary.sql"
+        if is_managed_ocp_cloud_summary_enabled(self.schema, Provider.PROVIDER_AWS):
+            sql_file = "trino_sql/aws/openshift/managed_reporting_ocpawscostlineitem_daily_summary.sql"
```

Comment:
I have been making an effort to organize the sql files for populating the ocp on cloud daily summary table under the `daily_summary_flow` directory. We should consider moving these files under that directory to align with the files needed to populate this table.

Reply:
After a discussion with Luke he convinced me that if we had a `transition_to_posgresql` folder that these files would make more sense in there then what we are currently calling the  `daily_summary_flow`.